src/main/java/jenkins/plugins/slack/ActiveNotifier.java

```diff
@@ -102,26 +101,6 @@ private void notifyStart(AbstractBuild build, String message) {
     }
 
     public void finalized(AbstractBuild r) {
```

Comment:
not sure why this implements an interface that we own.

What's the point of FineGrainedNotifier?

Reply:
this has existed since initial import from hipchat plugin in 2013, a.k.a no idea